src/ETISS.cpp

```diff
@@ -482,69 +484,85 @@ void etiss_loadIniConfigs()
-                if (std::string{ iter_section.pItem } == "StringConfigurations")
+                // get all values of a key with multiple values
+                CSimpleIniA::TNamesDepend values;
+                po_simpleIni->GetAllValues(iter_section.pItem, iter_key.pItem, values);
+                for (auto iter_value : values)
                 {
-                    message << "   Multi values. Take only LAST one!";
+                    // Handle configurations
+                    if (std::string(iter_section.pItem) == "StringConfigurations")
+                    {
+                        etiss::cfg().set<std::string>(iter_key.pItem, iter_value.pItem);
+                    }
+                    else if (std::string(iter_section.pItem) == "BoolConfigurations")
+                    {
+                        etiss::cfg().set<bool>(iter_key.pItem,
+                                            po_simpleIni->GetBoolValue(iter_section.pItem, iter_key.pItem));
+                    }
+                    else if (std::string(iter_section.pItem) == "IntConfigurations") // already load!
+                    {
+                        etiss::cfg().set<long long>(iter_key.pItem,
+                                                    po_simpleIni->GetDoubleValue(iter_section.pItem, iter_key.pItem));
+                        // we use double, as long could have only 32 Bit (e.g. on Windows)
+                        // and long long is not offered by the ini library
+                    }
+                    else
+                    // we don't add a DoubleConfigurations section, as converting them
+                    // to and from strings could provoke accuracy issues.
+                    // To support double, a Configuration::get<double>() has to be
+                    // added to Misc.cpp
+                    {
+                        message << "   Section not found  for Value:";
+                        warning = true;
+                    }
+
+                    // write item (=option value) to message.
+                    message << iter_value.pItem << ",";
                 }
-                else
+
+                // check if more than one value is set in the ini file
+                if (values.size() > 1)
                 {
-                    message << "   Multi values. Take only FIRST one!";
+                    warning = true;
+                    if (std::string{ iter_section.pItem } == "StringConfigurations")
+                    {
+                        message << "   Multi values. Take only LAST one!";
+                    }
+                    else
+                    {
+                        message << "   Multi values. Take only FIRST one!";
+                    }
                 }
             }
-
             // add message to etiss log.
             etiss::log(warning ? etiss::WARNING : etiss::INFO, message.str());
         }
     }
 }
 
+bool set_cmd_check(std::string cmdlineoption)
```

Comment:
a function that is only used in the translation unit where it is defined, should be marked `static`, so that the symbol is not available to other translation units